replay/src/main.rs

```diff
@@ -120,7 +120,6 @@ fn main() {
         debug: false,
         enable_subpixel_aa: false,
         clear_framebuffer: true,
-        clear_empty_tiles: false,
```

Comment:
Since we have to clear the z-buffer now, I figure we might as well just do a clear of the color buffer as well. This also makes it perform much better on mobile / tiled GPUs which rely on a clear due to the way tiling works. We could definitely re-instate this later as an optimization if it makes sense to.